### PR TITLE
Integrated new tendermint task

### PR DESCRIPTION
**Pull request by Lu191**
Added new task to **tendermint** node monitoring:
- **`TaskTendermintProposalVotingCheck`** check if the validator is voting to governance proposals (you need to specify the validator address in the configuration file)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ The install script can be customized with these flags:
 install --help
      --active-set <active_set_number> number of the validators in the active set (tendermint chain) [default is the number of active validators]
      --admin <@username> the admin telegram username that is interested to new governance proposals (tendermint)
- -a  --validator-address <address> enable checks on block production
+ -a  --validator-address <address> enable checks on block production (substrate) or checks on governance proposal votes (tendermint)
```
Comment (dakk) [inline README.md:95]: write "checks on block production governance proposals and other account related informations"

File: install.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ print_help () {
     echo "Usage: install [options...]
      --active-set <active_set_number> number of the validators in the active set (tendermint chain) [default is the number of active validators]
      --admin <@username> the admin telegram username that is interested to new governance proposals (tendermint)
- -a  --validator-address <address> enable checks on block production
+ -a  --validator-address <address> enable checks on block production (substrate) or checks on governance proposal votes (tendermint)
```
Comment (dakk) [inline install.sh:18]: same as before

File: srvcheck/chains/tendermint.py
Comment (dakk) [inline srvcheck/chains/tendermint.py:1]: why did you reverted all severity based changes?

File: srvcheck/chains/tendermint.py [hunk 3]
```diff
@@ -159,6 +140,50 @@ def run(self):
...
+        self.prev = None
+        self.validator_address = self.s.conf.getOrDefault("chain.validatorAddress")
+        self.serviceName = self.s.conf.getOrDefault("chain.service")
+
+    @staticmethod
+    def isPluggable(services):
+        return services.conf.exists("chain.validatorAddress") and services.conf.exists("chain.service")
+    
+    def getValidatorProposalVote(self, proposalId):
+        c = configparser.ConfigParser()
```
Comment (dakk) [inline srvcheck/chains/tendermint.py:155]: Move this code to a new method ```Chain.getNodeBinary```

**Review by dakk — CHANGES_REQUESTED** [review]

File: srvcheck/chains/tendermint.py
Comment (Lu191) [reply srvcheck/chains/tendermint.py:1]: Sorry, I was not using the latest version of srvcheck, now it should be ok

**Review by dakk — APPROVED** [review]

**Pull request merged**